features/db_effect.feature

```diff
@@ -0,0 +1,32 @@
+    Scenario: Data base effect implementation
+
+Feature: Data base effect validation
+        Given I get targets list
+        When I should see the 3 numerical targets in the list
+        And I should see each target number should be in the range of citizens list
+        And I should see only targets are relevant to alive citizens
+        Then validation method should return True
+
+Feature: Data base effect activation
+        Given activate method is called
+        Then active player should get message "Выберите три цели, которые будут проанализированы с помощью базы данных..."
+        And active player should enter the target number three times
+        When validation is successful
+        Then three targets should be appended into the effect target
+        When validation is failed
+        Then activation should be repeated
+
+Feature: Data base effect activated
+        Given resolve method is called and effect status is ACTIVATED
+        Then I should get relevant citizens types by targets numbers
+        When there are no Player or Spy among the citizens types
+        Then active player should get the message "База данных показала, что среди указанных граждан нет подозрительных личностей"
```

Comment:
Maybe, player 'that activated card', not the active player?

Reply:
Yes, my bad, i'll fix it, thanks)